code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -163,6 +163,13 @@
 		qdel(W)
 		qdel(src)
 
+obj/item/clothing/shoes/clown_shoes/lola
+	desc = "Honk!"
```

Comment:
>desc before name

This bothers my autism to no end.

Reply:
well the other clown shoes are like that so it's c o n s i s t e n t